product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx

```diff
@@ -30,8 +30,23 @@ Before connecting to your cloud,
       - bash shell version 4.0 or above.
 
       - [jq](https://stedolan.github.io/jq/) JSON parser.
+      
       - BigAnimal CLI version 1.4 or later. For detailed steps, see [Installing the CLI](../../reference/cli/#installing-the-cli).
 
+ - Be aware that BigAnimal requires permissions to run Kubernetes cluster services for PostgreSQL workloads and the associated storage services. It requires a set of supporting permissions:
+
+    - To log and monitor services to produce service telemetry information.
```

Comment:
Not a native speaker, so please disregard if I get this completely wrong. "To log and monitor services" sounds to me like BigAnimal is doing the action of logging and monitoring. While it's "services for monitoring and logging". It's a subtlety, I tried to convey the latter with the original take.

Reply:
You're right, I didn't understand what you meant on the first go around. I'll make that change.